addons/main/script_macros_common.hpp

```diff
@@ -936,6 +962,13 @@ Macro: PATHTO_FNC()
 Author:
     dixon13, commy2
  ------------------------------------------- */
+#ifdef RECOMPILE
```

Comment:
Now the PATHTO_FNC header looks like it applies to recompile.

Reply:
The alternative is having `RECOMPILE` to look like it belongs to `PREP` when it belongs to `PATHTO_FNC` only.

Reply:
maybe move it below PATHTO_FNC then? So that the header is still directly above what it belongs to

Reply:
`PATHTO_FNC` uses `RECOMPILE`. I think it would break the way you described.

Reply:
Macro inside a different macro is only resolved when the second macro is used.
PATHTO_FNC is resolved first. Then the RECOMPILE inside it is resolved.
So it wouldn't cause any problems if it's defined after the PATHTO_FNC macro